mono/metadata/appdomain.c

```diff
@@ -1840,9 +1840,11 @@ mono_make_shadow_copy (const char *filename, MonoError *oerror)
 	sibling_source_len = strlen (sibling_source);
 	sibling_target = g_strconcat (shadow, ".config", NULL);
 	sibling_target_len = strlen (sibling_target);
-	
+
 	copy_result = shadow_copy_sibling (sibling_source, sibling_source_len, ".mdb", sibling_target, sibling_target_len, 7);
 	if (copy_result)
```

Comment:
No, copy_result is whenever the copy succeeded, it succeeds even if the source doesn't exist.